loader/include/Geode/loader/Loader.hpp

```diff
@@ -94,17 +97,40 @@ namespace geode {
         bool hasLaunchArgument(std::string_view const name) const;
         /**
          * Get a launch argument. These are passed into the game as command-line arguments
-         * with the format `--geode:argName=value`.
+         * with the format `--geode:arg-name=value`.
          * @param name The argument name
          * @return The value, if present
          */
         std::optional<std::string> getLaunchArgument(std::string_view const name) const;
         /**
-         * Get a boolean launch argument. Returns whether the argument is present and its
+         * Get a launch argument flag. Returns whether the argument is present and its
          * value is exactly `true`.
          * @param name The argument name
          */
-        bool getLaunchBool(std::string_view const name) const;
+        bool getLaunchFlag(std::string_view const name) const;

```

Comment:
im worried about this causing undefined symbol errors in newer mods

Reply:
We've discussed this a bit in Discord -- presumably it shouldn't affect anyone